Doc/c-api/sys.rst

```diff
@@ -363,14 +356,23 @@ accessible to C code.  They all work with the current interpreter thread's
 
    .. audit-event:: sys.addaudithook "" c.PySys_AddAuditHook
 
-      If the interpreter is initialized, this function raises a auditing event
-      ``sys.addaudithook`` with no arguments. If any existing hooks raise an
-      exception derived from :class:`Exception`, the new hook will not be
-      added and the exception is cleared. As a result, callers cannot assume
-      that their hook has been added unless they control all existing hooks.
+   If the interpreter is initialized, this function raises an auditing event
+   ``sys.addaudithook`` with no arguments. If any existing hooks raise an
+   exception derived from :class:`Exception`, the new hook will not be
+   added and the exception is cleared. As a result, callers cannot assume
+   that their hook has been added unless they control all existing hooks.
 
    .. versionadded:: 3.8
 
+.. c:namespace:: NULL
+.. c:type:: int (*Py_AuditHookFunction) (const char *event, PyObject *args, void *userData)
```

Comment:
If it is not nested, `c:namespace` is not needed. I would prefer to make it nested, but it is only my personal preference.